hw/top_earlgrey/ip/sensor_ctrl/data/sensor_ctrl.hjson

```diff
@@ -142,12 +142,14 @@ registers: [
         hwaccess: "hro",
         regwen: "CFG_REGWEN",
         fields: [
-            { bits: "0",
+            { bits: "1:0",
               name: "VAL",
               desc: '''
                 Acknowledge mode to alerts.
                 0 is immediate acknowledge, the hardware immedaitely acks an incoming alert.
                 1 is software acknowledge.  Software must program the appropriate bit in !!ALERT_STATE to acknowledge the alert.
+                2 is no acknowledge.  When in this mode, any alert will effectively be considered fatal to the system.
```

Comment:
What is the purpose of this mode - more of a debug mode?

Reply:
you mean no ack? I think honestly no ack is going to be the preferred mode of operation.. it would mean that our sensors have no false positives, and we are comfortable leaving them in a mode where if they EVER fire, we treat it as fatal. 

The software ack mode is actually meant more as a debug mode to allow software to interject and figure out what's wrong.

Reply:
sounds good

Reply:
It's probably worth noting also that AST is the only block in the system where we've added some more alert flexibility, due to the analog nature of the alert sensors. All other IPs in the system should follow the `recov_`/`fatal_` guidance.